modules/veteran/app/models/veteran/service/organization.rb

```diff
@@ -36,6 +37,10 @@ def self.find_within_max_distance(long, lat, max_distance = DEFAULT_MAX_DISTANCE
       def self.find_with_name_similar_to(search_phrase)
         where('word_similarity(?, name) >= ?', search_phrase, FUZZY_SEARCH_THRESHOLD)
       end
+
+      def self.max_per_page
```

Comment:
Can we convert this class method in the `organizations` and `representative` models to a constant in `Veteran::Service::Constants`? Both methods return the same value.

Reply:
Done!

Reply:
Nice!